.github/workflows/build.yml

```
@@ -302,7 +302,7 @@ jobs:
         with:
           version: yarn changeset:version
           publish: yarn changeset:publish
-          commit: "Version Packages with 'latest' tag"
+          commit: "release: release Packages with 'latest' tag"
```

Still got this last one

```suggestion
          commit: "release: release packages with 'latest' tag"
```

---

Done!